src/test/scala/com/cognite/spark/datasource/EventsRelationTest.scala

```diff
@@ -96,7 +96,7 @@ class EventsRelationTest extends FlatSpec with Matchers with SparkTest {
                  |endTime,
                  |type,
                  |subtype,
-                 |null as assetIds,
+                 |array(2091657868296883) as assetIds,
```

Comment:
Maybe set it to "null" here, then change it in the "Update events" step, and then check the results to ensure they have the new value in the following tests?

Reply:
Good idea, I'll implement that.